src/spec-lang/tc/typecheck.ts

```diff
@@ -567,6 +587,86 @@ function isInty(type: SType): boolean {
     return type instanceof SIntType || type instanceof SIntLiteralType;
 }
 
+/**
+ * Sort `contracts` in topological order with respect to inheritance.
+ *
+ * @param contracts
```

Comment:
These line can be removed as do not provide anything useful.

Reply:
Fixed in dcf955c.